### ISSUE TITLE
[FEATURE] - Unhide USB page in Settings

### ISSUE THREAD

**Issue by hodifaalmosa** [Atlas-OS/Atlas#1104]
### Description

Hi, thanks for the perfect os,

I am on windows 10 x64 22h2 pro

And applied playbook atlas os

Problem is USB option not found in Bluetooth & devices

And Un checked disable notification, because it's important for my job,

To check usb devices if have issues when connecting to pc.

Although I have not disable notification the message "usb device not recognized" not appear

I have fix it by edit this value from 0 to 1

[HKEY_CURRENT_USER\Software\Microsoft\Shell\USB]

"NotifyOnUsbErrors"=dword:00000001

How do I fix it without the reedit / how to show usb option in Bluetooth & devices ??

### Steps to reproduce

. 

### Expected behavior

. 

### Actual behavior

. 

### Atlas Edition

Atlas for Windows 10 22H2

### Desktop information

.

**Comment by hodifaalmosa** [comment]
I fixed it

Regedit > hkey local machine, software, Microsoft, Windows, current version, polices, Explorer

Settingspagevisibility 

Remove :usb

**Comment by Ast3risk-ops** [comment]
@JayXTQ closde (solved + not filled out right)

**Comment by he3als** [comment]
Hi, please fill out the form correctly next time. Your issue will be fixed next release

**Comment by he3als** [comment]
For anyone else looking at this issue, the easiest fix is to:

1. Open Command Prompt as an Administrator
2. Copy, paste and press **Enter** on:
```bat
call "%windir%\AtlasModules\Scripts\settingsPages.cmd" /unhide USB
```

**Comment by he3als** [comment]
Fixed in [release v0.4.1](https://github.com/Atlas-OS/Atlas/releases/tag/0.4.1). See the release for more info on what v0.4 users can do.
